src/components/useCase/UseCaseHead/index.ts

```diff
@@ -0,0 +1 @@
+export { default as UseCaseHead } from './Component.astro';
```

Comment:
![Screenshot 2024-10-30 at 15 25 21](https://github.com/user-attachments/assets/0b86d015-329d-4c84-a1c5-c184a94fbbe2)

Secondo me tutti questi è meglio metterli in una cartella pages/customers/[slug]/_sub, come ho fatto per src/pages/pricing/_sub/.. così non inquiniamo components con cose usate in una sola pagina.

Il discorso vale ovviamente per cose simili in altre pagine.